.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 
 jobs:
   default:
-    name: Build and Test on ${{ matrix.os }} with Node.js ${{ matrix.node }}
+    name: Test on ${{ matrix.os }} with Node.js ${{ matrix.node }}
```

Comment:
:man_shrugging: it still does build and test https://github.com/mdx-js/mdx/blob/a95e0dd7edcfe9edcdbb8ee5196e2ac9ae91e41e/package.json#L32

Reply:
😂 This is unobtrusively in the workflow.

Reply:
Can you explain? I tend to agree with @ChristianMurphy

Reply:
I mean in the workflow file, there is only `yarn test`. `build` is implicit in `package.json`.

Of course, I can add it back if you want.

Reply:
It doesn’t matter much now... But I would’ve preferred it separately or not at all 🤷‍♂️

Reply:
I'd prefer separate too. It actually should be `pretest: yarn build` I think.